pinot-core/src/main/java/org/apache/pinot/core/transport/ServerRoutingInstance.java

```diff
@@ -23,43 +23,36 @@
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 import javax.annotation.concurrent.ThreadSafe;
-import org.apache.pinot.spi.config.table.TableType;
 import org.apache.pinot.spi.utils.CommonConstants.Helix;
 
 
 /**
  * The {@code ServerRoutingInstance} class represents the routing target instance which contains the information of
- * hostname, port, and table type it serves.
- * <p>Different table types on same host and port are counted as different instances. Therefore, one single Pinot Server
- * might be treated as two different routing target instances based on the types of table it serves.
+ * hostname and port.
  */
 @ThreadSafe
 public final class ServerRoutingInstance {
-  private static final String SHORT_OFFLINE_SUFFIX = "_O";
```

Comment:
I think with this change we kind of lose the way of knowing how many servers get queried/respond for each table type. I understand that the number of queries being hit to the same server can be reduced, but is it possible to include this table type information in the request and response, so that we don't lose this stat?

Reply:
Linking comments:  https://github.com/apache/pinot/pull/13742/files#r1716084206